model/MessageQueue.go

```diff
@@ -0,0 +1,50 @@
+package model
+
+import (
+	"container/heap"
+	"sync"
+
+	"github.com/puchkovki/distr-model/model"
+	"github.com/puchkovki/distr-model/priorityq"
```

Comment:
Change puchkovki to trmigor.
Delete "github.com/puchkovki/distr-model/model"

Reply:
How? Your repo doesn't include `priorityq` package.

Reply:
It will. After I merge PR.